src/common/transformations/include/ov_ops/swiglu.hpp

```diff
@@ -5,22 +5,19 @@
 #pragma once
 
 #include "openvino/op/op.hpp"
+#include "transformations_visibility.hpp"
 
 namespace ov {
-namespace intel_gpu {
 namespace op {
+namespace internal {
 
 /// \brief Operator performing Swish Gated Linear Unit Activation
 /// This operation performs gated linear unit activation that combines swish or gelu activation function
-class SwiGLU : public ov::op::Op {
+class TRANSFORMATIONS_API SwiGLU : public ov::op::Op {
 public:
-    OPENVINO_OP("SwiGLU", "gpu_opset");
+    OPENVINO_OP("SwiGLU", "ie_internal_opset");
```

Comment:
Actually, it is "SwiGLU" only when the activation is swish. 
If the activatin is gelu, it is "GeGLU". 
In gpu plugin, we added Swiglu fisrst, and then, we just did not make effort to update the op name when we newly met Geglu. 
However, if we are going to add a common internal op, how about to change the name as GLU with types of swish, gelu?

Reply:
I've shared the same idea within the ticket (155542), so I agree the name could be aligned. 
> (Optional) Rename SwiGLU to more generic name like FuncGLU to avoid confusion

But to keep it simple and don't mix the changes, I would prefer to do it in the following order:
1. Move the SwiGLU op as is
2. Rename the op

Reply:
PR renaming SwiGLU to GLU: 
- https://github.com/openvinotoolkit/openvino/pull/27683